addon/components/import-snippet-card.js

```diff
@@ -0,0 +1,81 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+export default class EditorPluginsImportAsAttachmentComponent extends Component {
+  @service importRdfaSnippet;
+  @tracked snippets = [];
+  @tracked besluitNode;
+
+  constructor() {
+    super(...arguments);
+    this.args.controller.onEvent(
+      'selectionChanged',
+      this.selectionChangedHandler
+    );
+  }
+
+  @action
+  selectionChangedHandler() {
+    this.snippets = this.importRdfaSnippet.snippetsForType('roadsign');
+    const limitedDatastore = this.args.controller.datastore.limitToRange(
+      this.args.controller.selection.lastRange,
+      'rangeIsInside'
+    );
+    const besluit = limitedDatastore
+      .match(null, 'a', '>http://data.vlaanderen.be/ns/besluit#Besluit')
+      .asSubjectNodes()
+      .next().value;
+    if (besluit) {
+      this.besluitNode = [...besluit.nodes][0];
+    } else {
+      this.besluitNode = undefined;
+    }
+  }
+
+  @action
+  insert(snippet, type) {
+    const html = this.generateSnippetHtml(snippet, type);
+    let rangeToInsert;
+    if (this.besluitNode) {
+      rangeToInsert = this.args.controller.rangeFactory.fromInNode(
+        this.besluitNode,
+        this.besluitNode.getMaxOffset(),
+        this.besluitNode.getMaxOffset()
+      );
+    } else {
+      rangeToInsert = this.args.controller.selection.lastRange;
+    }
+    this.args.controller.executeCommand('insert-html', html, rangeToInsert);
+    this.importRdfaSnippet.removeSnippet(snippet);
+  }
+
+  generateSnippetHtml(snippet, type) {
+    if (type === 'attachment') {
+      return `
+        <div 
+          resource="${snippet.source}" 
+          property="http://lblod.data.gift/vocabularies/editor/isLumpNode http://data.europa.eu/eli/ontology#cites"
```

Comment:
```suggestion
          property="http://lblod.data.gift/vocabularies/editor/isLumpNode http://data.europa.eu/eli/ontology#related_to"
```
we should use eli:related_to instead of eli:cites

Reply:
actually, looking at this again this should be changed so the related_to property is nested inside the lumpNode